modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * install_config/installing-operator-framework.adoc
+
+[id='uninstalling-olm-using-ansible_{context}']
+= Uninstalling Operator Lifecycle Manager using Ansible
```

Comment:
The contributor guides state to use sentence case for all module titles/section headings. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

Reply:
This is probably right because the individual components need to be capitalized correctly.

Reply:
Ack

Reply:
3.11 actually uses title case (we never changed that). See the 3.11-specific guidelines here: https://github.com/openshift/openshift-docs/blob/enterprise-3.11/contributing_to_docs/doc_guidelines.adoc#topic-titles-and-section-headings

There are also some other 3.11-specific differences. I should probably add this to the peer review onboarding docs

CC @kalexand-rh @opayne1 too